tests/common/devices/multi_asic.py

```diff
@@ -21,7 +21,7 @@ class MultiAsicSonicHost(object):
     So, even a single asic pizza box is represented as a MultiAsicSonicHost with 1 SonicAsic.
     """
 
-    _DEFAULT_SERVICES = ["pmon", "snmp", "lldp", "database"]
+    _DEFAULT_SERVICES = ["pmon", "snmp", "database"]
 
```

Comment:
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/devices/sonic.py#L43
and line #53 may also need to be fixed. Please check.

Reply:
It is not needed here. lldp still applies to Default asic service list